### ISSUE TITLE
Physics deleting items on land

### ISSUE THREAD

**Issue by thislooksfun** [EnviroMine/EnviroMine-1.7#123]
Title says it all.

**Comment by glixyl** [comment]
When a block moves it removes metadata, is that what you are speaking about?  If so, try setting physics to average or strong for blocks > id 175 such as machines; within the enviromine config.  Gravestones are typically problematic when used with enviromine.

**Comment by thislooksfun** [comment]
No, someone reported items on the ground being deleted when a physics block lands on them.
Also, what metadata are you talking about? Because that sounds like a bug too.
